core/src/banking_stage.rs

```diff
@@ -390,7 +390,8 @@ impl BankingStage {
                         ),
                     };
 
-                let mut packet_receiver = PacketReceiver::new(id, packet_receiver);
+                let mut packet_receiver =
+                    PacketReceiver::new(id, packet_receiver, bank_forks.clone());
```

Comment:
Passing `bank_forks` to `PacketReceiver`, so packets can be flagged with root_bank's feature set.